docs/guides/admin/docs/configuration/database.md

```diff
@@ -104,49 +103,80 @@ Finally, leave the client and restart the database server to enable the new user
 % systemctl restart mariadb.service
 ```
 
+PostgreSQL
+----------
 
-### Set up the Database Structure
+Opencast's official PostgreSQL support is still marked as experimental.
 
-To set up the database structure you can (and should!) use the Opencast ddl scripts. You can find them in
-`…/docs/scripts/ddl/mysql5.sql` or download them from GitHub.
+Install PostgreSQL, create a database and a user:
 
-To import the database structure using the MariaDB client, switch to the directory that contains the `mysql5.sql` file,
-run the client with a user privileged to create the database structure and switch to the database you want to use
-(e.g. `opencast`):
-
-```sh
-% mysql -u root -p opencast
 ```
-
-Run the ddl script:
-
-```
-mysql> source mysql5.sql;
+sudo -u postgres psql
+postgres=# create database opencast;
+postgres=# create user opencast with encrypted password 'opencast_password';
```

Comment:
Maybe it is helpful to add a comment that by default PostgreSQL uses IDENT-based authentication, thus username and password authorization has to be configured.

Reply:
Depends on your installation type. E.g. containers use password by default. But I'll add a comment